src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java

```diff
@@ -86,11 +95,9 @@ public void verifyLightboxVideo() {
     logMessage(GREEN_OOYALA_3, VIDEO_DURATION_SEC);
   }
 
-  private void verifyFeaturedVideoElement(By selector, Color color, int duration) {
-    wait.forElementVisible(selector, 30, 1000);
-    scrollToPosition(ARTICLE_VIDEO_WRAPPER_SELECTOR);
-    fixScrollPositionByNavbar();
-    verifyColorAd(articleVideoWrapper, color, 5);
+  private void verifyFeaturedVideoElement(Color color, int duration) {
+//    verifyColorAd(articleVideoWrapper, color, 5);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)